get-dependencies.sh

```diff
@@ -0,0 +1,42 @@
+apt-get install \
+gconf-service \
+libasound2 \
+libatk1.0-0 \
+libatk-bridge2.0-0 \
+libc6 \
+libcairo2 \
+libcups2 \
+libdbus-1-3 \
+libexpat1 \
+libfontconfig1 \
+libgcc1 \
+libgconf-2-4 \
+libgdk-pixbuf2.0-0 \
+libglib2.0-0 \
+libgtk-3-0 \
+libnspr4 \
+libpango-1.0-0 \
+libpangocairo-1.0-0 \
+libstdc++6 \
+libx11-6 \
+libx11-xcb1 \
+libxcb1 \
+libxcomposite1 \
+libxcursor1 \
+libxdamage1 \
+libxext6 \
+libxfixes3 \
+libxi6 \
+libxrandr2 \
+libxrender1 \
+libxss1 \
+libxtst6 \
+ca-certificates \
+fonts-liberation \
+libappindicator1 \
+libnss3 \
+lsb-release \
+xdg-utils \
+wget
+
+yarn add puppeteer
```

Comment:
When installing inside of pioneer folder, we actually need `yarn add puppeteer -W` to install in root workspace.
I think if we add proper dependencies to package.json the package can be installed in its own folder.